ansible/roles/cri/defaults/main.yml

```diff
@@ -1,2 +1,4 @@
 ---
 cri_plugin: docker
+docker_cache_enabled: false
+cache_registry_fqdn: ""
```

Comment:
I was wondering if these should exist in the same group_vars rather than being split into separate group var locations?

Reply:
the `cache_registry_fqdn` can likely be removed.

if we don't define `docker_cache_enable` (i fixed the typo by the way) with a default value the role will not function. it _is_ up to the user to decide if they want the cache or not, but that should be determined at the group_vars level. if the user doesn't provide that var, it should default on the CRI role as `false` to avoid accidentally configuring the docker daemon to point at a registry mirror that doesn't exist.

granted, this will result in an ansible error anyways, but from a UX perspective i believe that this is the correct action to take.

for `cache_registry_fqdn` i have removed it from this file.